modules/gmc-agent/src/main/java/org/gridgain/dto/topology/TopologySnapshot.java

```diff
@@ -17,13 +17,15 @@
 package org.gridgain.dto.topology;
 
 import java.util.Collection;
-import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
 import org.apache.ignite.cluster.BaselineNode;
 import org.apache.ignite.cluster.ClusterNode;
 import org.apache.ignite.internal.util.typedef.internal.S;
 
 import static java.util.Collections.emptyList;
-import static java.util.stream.Collectors.toList;
+import static java.util.stream.Collectors.*;
```

Comment:
Wild card imports are not allowed. Configure IDEA settings.

Reply:
Ooops,